versioned_docs/version-5.x/screen-tracking.md

```diff
@@ -13,12 +13,9 @@ To get notified of state changes, we can use the `onStateChange` prop on `Naviga
 
 ## Example
 
-This example shows how to do screen tracking and send to Firebase Analytics using [expo-firebase-analytics](https://docs.expo.io/versions/latest/sdk/firebase-analytics/). The approach can be adapted to any other analytics SDK.
-
- <samp id="screen-tracking-for-analytics" />
```

Comment:
Can you add this back here too?

Reply:
I missed that. Sorry, my bad 🙏 

Added it back.